Sources/Sentry/SentryUIEventTracker.m

```diff
@@ -84,6 +85,12 @@ - (void)start
 
             [currentActiveTransaction finish];
 
+            [SentryLog
+                logWithMessage:[NSString
+                                   stringWithFormat:@"SentryUIEventTracker finished transaction %@",
+                                   currentActiveTransaction.transactionContext.name]
+                      andLevel:kSentryLevelDebug];
+
```

Comment:
m: I believe `currentActiveTransaction` may be null and no transaction was finished

Reply:
Are you sure? Looking at the code I see no indication that it should be able to be null. Oh how I miss Swift at times like these 😅

Reply:
Its very likely to not have a transaction going on.

Reply:
Ok, added the check to #2093